samples/azure-service-bus-netstandard/key-vault/ASBS_6/Shared/KeyVaultBasedConfigurationProvider.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+
+namespace Shared
+{
+    #region config
```

Comment:
I would narrow down the region to `GetConfiguration` method only and pulled the NuGet comments out of there.